### ISSUE TITLE
Plugin Not Found

### ISSUE THREAD

**Issue by PCTipsGR** [CodeSequence/sanmyaku#56]
Issue:

> The addon could not be enabled because it generated a fatal error:
> Plugin not found.

Vanilla Version Running:
**2.6.1**

Expected Behaviour:
Theme to display itself

Installing Procedure Followed:
Downloaded theme from https://open.vanillaforums.com/addon/sanmyaku-theme and pasted to the themes folder of my Vanilla installation.

**Comment by MicrosoftMCreates** [comment]
I am having this issue aswell on the same forum version.

**Comment by PCTipsGR** [comment]
I actually fixed that by renaming the sanmyaku-1.0.3 folder to sanmyaku @MicrosoftMCreates, however it crashed my Vanilla installation and it still is not working.

I hope that the devs will review and fix this soon.

**Comment by valeryan** [comment]
Hey guys I have moved on from using vanilla currently and I don't have any means of testing and developing this theme. I would assume your problems is because the api for loading smarty plugins has changed slightly. If you want to do a PR that fixes it we will merge it but I am not even sure this project still builds if we updated it. We have added a badge to our readme showing maintainers wanted. Feel free to volunteer.
